plugin/trino-delta-lake/src/test/resources/databricks/json_stats_on_row_type/_delta_log/_last_checkpoint

```diff
@@ -0,0 +1 @@
+{"version":2,"size":4}
```

Comment:
for the test, do we need transaction json files before the checkpoint (0 and 1) ?

Reply:
Those files aren't required. Removed.